var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -137,9 +139,11 @@ class Gdal(CMakePackage, AutotoolsPackage, PythonExtension):
     variant("kdu", default=False, description="Required for JP2KAK and JPIPKAK drivers")
     variant("kea", default=False, description="Required for KEA driver")
     variant("lerc", default=False, when="@2.4:", description="Required for LERC compression")
+    variant("libaec", default=False, when="@3.8:", description="Optional for GRIB driver")
     variant("libcsf", default=False, description="Required for PCRaster driver")
     variant("libkml", default=False, description="Required for LIBKML driver")
     variant("liblzma", default=False, description="Required for Zarr driver")
+    variant("libqb3", default=False, when="@3.6:", description="Required for MRF driver")
```

Comment:
Curious.  Is this a good time to introduce the `requires()` directive since there are multiple variants that appear to be required for the `MRF` driver, for example?  Or conflicts if all required together are not provided?

Docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#conflicts-and-requirements.

Reply:
Probably not in GDAL. Many of these deps are used by multiple drivers and may or may not actually be strictly required. But I can think of a few other packages like OpenCV and PyTorch where `requires()` will be helpful.